BlueDBOnDisk/src/main/java/io/bluedb/disk/collection/CollectionMetaData.java

```diff
@@ -15,13 +16,15 @@
 	private static final String FILENAME_MAX_INTEGER = "max_int";
 	private static final String FILENAME_MAX_LONG = "max_long";
 	private static final String FILENAME_SERIALIZED_CLASSES = "serialized_classes";
+	private static final String FILENAME_KEY_TYPE = "key_type";
 	private static final String META_DATA_FOLDER = ".meta";
 	
 	final Path folderPath;
 	final FileManager fileManager;
 	final Path maxIntegerPath;
 	final Path maxLongPath;
 	final Path serializedClassesPath;
+	final Path keyTypePath;

```

Comment:
You could generate these paths on demand right? We don't really need to have the cached all the time?

Reply:
Yeah.  Because some of these are accessed frequently, I calculated their paths up front (also makes methods easier to read).  However, it's not really necessary to cache them.  On the other hand, it's one per collection so it doesn't take much memory.  I don't really care strongly either way.